lib/bakeAmbientOcclusion.js

```diff
@@ -39,6 +39,34 @@ var cartesian2Scratch = new Cartesian2();
 var quaternionScratch = new Quaternion();
 var matrix3Scratch = new Matrix3();
 var matrix4Scratch = new Matrix4();
+var trianglePixelMarchScratch = {
+    uv0 : cartesian2Scratch,
+    uv1 : cartesian2Scratch,
+    uv2 : cartesian2Scratch,
+    position0 : worldPositionScratch,
+    position1 : worldPositionScratch,
+    position2 : worldPositionScratch,
+    normal0 : worldNormalScratch,
+    normal1 : worldNormalScratch,
+    normal2 : worldNormalScratch,
+    transform : matrix4Scratch,
+    inverseTranspose : matrix4Scratch,
+    raytracerScene : {},
+    pixelWidth : 0.0,
+    resolution : 0,
+    callback : {},
+    parameters : {}
+};
+
+var computeAmbientOcclusionAtScratch = {
+    position : worldPositionScratch,
```

Comment:
Should I switch these two (`computeAmbientOcclusion` and `trianglePixelMarch`) back to just using arguments? From the video it seems like inline caching would speed up property access but since that would require a constructor function we would lose the self documentation.